src/Mutation/Item.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Softonic\GraphQL\Mutation;
+
+use Softonic\GraphQL\Traits\JsonSerializer;
+
+class Item implements MutationObject, \JsonSerializable
+{
+    use JsonSerializer;
+
+    private $arguments;
+
+    private $config;
+
+    private $hasChanged = false;
+
+    public function __construct(array $arguments = [], array $config = [], bool $hasChanged = false)
+    {
+        $this->arguments  = $arguments;
+        $this->config     = $config;
+        $this->hasChanged = $hasChanged;
+    }
+
+    public function __get(string $key)
+    {
+        if (is_null($this->arguments[$key]) && array_key_exists($key, $this->config)) {
+            $mutationType = $this->config[$key]->type;
+
+            $this->arguments[$key] = new $mutationType([], $this->config[$key]->children);
+        }
+
+        return $this->arguments[$key];
+    }
+
+    public function __set(string $key, $value)
+    {
+        $this->arguments[$key] = $value;
+
+        $this->hasChanged = true;
```

Comment:
We can have a good improvement in bulk actions if we just change the "hasChanged" stated when it really changes.

POC

```
public function __set(string $key, $value)
{
    if ($this->arguments[$key] !== $value) {
         $this->arguments[$key] = $value;
         $this->hasChanged = true;
     }
}
```